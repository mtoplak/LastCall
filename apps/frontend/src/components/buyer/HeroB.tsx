import { Box, Typography, Container } from '@mui/material';
import heroImg from '../../assets/images/homepageDrink.png';
import CustomBox from 'components/ui/CustomBox';
import Title from 'components/ui/Title';

const Hero = () => {
	return (
		<Box sx={{ backgroundColor: '#E6F0FF', minHeight: '65vh' }}>
			<Container>
				<CustomBox>
					<Box sx={{ flex: '1' }}>
						<Typography
							variant="body2"
							sx={{
								fontSize: '18px',
								color: '#687690',
								fontWeight: '500',
								mt: 10,
								mb: 4,
							}}
						>
							Welcome to LastCall
						</Typography>
						<Title variant="h1">
							Bridging the Gap Between Drink{' '}
							<span style={{ color: '#24336e' }}>Buyers</span> and
							Sellers
						</Title>
					</Box>

					<Box sx={{ flex: '1.25' }}>
						<img
							src={heroImg}
							alt="heroImg"
							style={{ maxWidth: '100%', marginTop: '5rem' }}
						/>
					</Box>
				</CustomBox>
			</Container>
		</Box>
	);
};

export default Hero;
