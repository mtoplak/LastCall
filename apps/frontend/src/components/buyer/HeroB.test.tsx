import { render } from '@testing-library/react';
import HeroB from './HeroB';

describe('Hero component', () => {
	it('should render', () => {
		const { container } = render(<HeroB />);
		expect(container).toMatchSnapshot();
	});
});