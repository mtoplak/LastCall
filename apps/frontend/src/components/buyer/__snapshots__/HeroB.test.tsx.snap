// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component should render 1`] = `
<div>
  <div
    class="MuiBox-root css-ff6udn"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-y2rtej"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-lg-6 css-ogcrwo-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-xi606m"
            >
              <p
                class="MuiTypography-root MuiTypography-body2 css-7irn93-MuiTypography-root"
              >
                Welcome to LastCall
              </p>
              <h1
                class="MuiTypography-root MuiTypography-h1 css-1qd72hw-MuiTypography-root"
              >
                Bridging the Gap Between Drink
                 
                <span
                  style="color: rgb(36, 51, 110);"
                >
                  Buyers
                </span>
                 
                and Sellers
              </h1>
              <p
                class="MuiTypography-root MuiTypography-body2 css-16rwcaa-MuiTypography-root"
              >
                With an unwavering commitment to enhancing the beverage industry, we unite suppliers, distributors, retailers, and enthusiastic consumers.
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-lg-6 css-f0xsrb-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-y5loaw"
            >
              <img
                alt="heroImg"
                src="homepageDrink.png"
                style="width: 100%; margin-top: 0px;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
